lib/hanami/action.rb

```diff
@@ -417,9 +417,10 @@ def _requires_no_body?(res)
     # @since 2.0.0
     # @api private
     def enforce_accepted_mime_types(request)
-      return unless config.accepted_formats.any?
+      return if config.accepted_formats.none?
```

Comment:
```suggestion
      return if config.accepted_formats.empty?
```

this is faster

Reply:
Thanks, @solnic! Fixed, and I'll remember this for next time :)